README.md

```diff
@@ -412,14 +412,56 @@ DeploymentApi api = new DeploymentApi(client);
 
 For more customization, creating a [HeaderProvider](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/http-destinations#about-headerproviders) is also possible.
 
-## Requirements and Setup
+## Requirements and Setup for AI Core
 
-### Set AI Core credentials as environment variable
+For any AI Core service interaction, the SAP Cloud SDK for AI requires credentials to be available at application runtime.
+By default, the credentials are extracted automatically from a service instance of type "aicore" bound to the application. 
+Running the application locally without such service binding will throw an exception:
 
-- Running the application locally without a service binding will throw:
+```
+Could not find any matching service bindings for service identifier 'aicore'
+```
+
+There are multiple options to register the service binding:
+* Regular service binding in SAP BTP Cloud Foundry (resulting in `VCAP_SERVICES` env var entry).
+* Set an environment variable explicitly: `AICORE_SERVICE_KEY`
```

Comment:
```suggestion
* Regular service binding in SAP BTP Cloud Foundry (resulting in `VCAP_SERVICES` env var entry).
* For **CAP applications** use the [hybrid testing](https://cap.cloud.sap/docs/advanced/hybrid-testing#services-on-cloud-foundry) approach. For example: `cds bind --to aicore --exec mvn spring-boot:run`
* Set an environment variable explicitly: `AICORE_SERVICE_KEY`
```

Reply:
I slightly rephrased your suggestion and put below env var